### ISSUE TITLE
registry-creds does not hide password when typing

### ISSUE THREAD

**Issue by justechn** [kubernetes/minikube#1608]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Feature Request

**Minikube version** (use `minikube version`):
0.19.1

**Environment**:
- **OS** (e.g. from /etc/os-release): macos
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`):
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`):
- **Install tools**:
- **Others**:

**What happened**:
When using the registry-creds addon the password is not hidden
```
$ minikube addons configure registry-creds

Do you want to enable AWS Elastic Container Registry? [y/n]: n

Do you want to enable Google Container Registry? [y/n]: n

Do you want to enable Docker Registry? [y/n]: y
-- Enter docker registry server url: dockerregistry.com
-- Enter docker registry username: myusername
-- Enter docker registry password: mypassword
registry-creds was successfully configured
```

**What you expected to happen**:
the password should be hidden

**How to reproduce it** (as minimally and precisely as possible):
configure the registry creds and type a password

**Anything else do we need to know**:

**Comment by r2d4** [comment]
I'll mark this as a bug, it should hide your password while reading it.

Maybe we could use `ReadPassword` from the `crypto/ssh/terminal package`, 
https://godoc.org/golang.org/x/crypto/ssh/terminal#Terminal.ReadPassword

**Comment by r2d4** [comment]
cc @abbytiz might be a fun bug to tackle, otherwise me or @aaron-prindle could get it

Basically, I think we need another function like `AskForStaticValue`, maybe something like `AskForPasswordValue`

See: 

https://github.com/kubernetes/minikube/blob/9b539db1ff2d08fcae34e13b465ee0ea58af5b8b/cmd/minikube/cmd/config/prompt.go#L56

https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/config/configure.go#L82

**Comment by r2d4** [comment]
Fixed by #1708
